Dungeoneer/src/com/interrupt/dungeoneer/entities/Monster.java

```diff
@@ -289,19 +289,22 @@
 	public float spawnMomentumTransfer = 1.0f;
 
 	private transient Entity attackTarget = null;
+    /** Sound played while monster is idle. */
+    @EditorProperty
+    private Float viewDistance = 17.0f;
```

Comment:
How about you use the type `float` instead of the object `Float`?

Reply:
Also, it would be great if you could add this to the JSON schema `jsonschema\current\dungeoneer\entities\Monster.schema.json`.